docs/source/posts/2023/2023-08-25-final-report-praneeth.rst

```diff
@@ -0,0 +1,215 @@
+
+	Based on these findings, we have decided to close all related issues and pull requests for now. If the need arises in the future, we can revisit this topic.
+	
+	**Topic:** - https://github.com/fury-gl/fury/discussions/816
+
+
+Other Objectives
+----------------
+
+- **Reviewing & Merging:**
+	In this phase, my focus was not on specific coding but rather on facilitating the completion of ongoing PRs. Here are two instances where I played a role:
+
+	1. **CardUI PR:**
+	   I assisted with the ``CardUI`` PR by aiding in the rebase process and reviewing the changes. The CardUI is a simple UI element consisting of an image and a description, designed to function like a flash card. I worked closely with my mentor to ensure a smooth rebase and review process.
+
+	2. **ComboBox Issue:**
+	   There was an issue with the ``ComboBox2D`` functionality, where adding it to a ``TabUI`` caused all elements to open simultaneously, which shouldn't be the case. I tested various PRs addressing this problem and identified a suitable solution. I then helped the lead in reviewing the PR that fixed the issue, which was successfully merged.
+
+	**Pull Requests:**
+  - **CardUI (Merged)** - https://github.com/fury-gl/fury/pull/398
+  - **ComboBox Flaw (Merged)** - https://github.com/fury-gl/fury/pull/768
+	
+
+    .. image:: https://user-images.githubusercontent.com/54466356/112532305-b090ef80-8dce-11eb-90a0-8d06eed55993.png
+        :height: 500
+        :align: center
+        :alt: CardUI
+	
+
+- **Updating Broken Website Links:**
+	I addressed an issue with malfunctioning links in the Scientific Section of the website. The problem emerged from alterations introduced in PR `#769 <https://github.com/fury-gl/fury/pull/769>`_. These changes consolidated demos and examples into a unified "auto_examples" folder, and a toml file was utilized to retrieve this data and construct examples. However, this led to challenges with the paths employed in website generation. My responsibility was to rectify these links, ensuring they accurately direct users to the intended content.
+
+	**Pull Requests:**
+  - **Updating Broken Links (Merged)** - https://github.com/fury-gl/fury/pull/820
+
+
+Objectives in Progress
+----------------------
+
+- **FileDialogUI:**
+	An existing ``FileDialog`` PR by Soham (`#294 <https://github.com/fury-gl/fury/pull/294`_`) was worked upon. The primary task was to rebase the PR to match the current UI structure, resolving compatibility concerns with the older base. In PR `#832 <https://github.com/fury-gl/fury/pull/832>`_, we detailed issues encompassing resizing ``FileDialog`` and components, addressing text overflow, fixing ZeroDivisionError, and correcting ``ListBox2D`` item positioning. The PR is complete with comprehensive testing and documentation. Presently, it's undergoing review, and upon approval, it will be prepared for integration.
```

Comment:
link for #294 is not building, a `>` is missing in the end, and I guess it would be good to turn "ZeroDivisionError" to "*ZeroDivisionError*"